src/views/MarketsDropdown.tsx

```diff
@@ -85,15 +89,15 @@ const MarketsDropdownContent = ({
               {isNew && <Tag isHighlighted>{stringGetter({ key: STRING_KEYS.NEW })}</Tag>}
             </$MarketName>
           ),
-        },
+        } satisfies ColumnDef<MarketData>,
         {
           columnKey: 'oraclePrice',
           getCellValue: (row) => row.oraclePrice,
           label: stringGetter({ key: STRING_KEYS.PRICE }),
           renderCell: ({ oraclePrice, tickSizeDecimals }) => (
             <$Output type={OutputType.Fiat} value={oraclePrice} fractionDigits={tickSizeDecimals} />
           ),
-        },
+        } satisfies ColumnDef<MarketData>,
```

Comment:
could we potentially add this at the end and do something like `satisfies ColumnDef<MarketData>[]`?